### DISCUSSION TITLE
Webdriver execution does not star.

### DISCUSSION THREAD

**Discussion by CSanchez58** [webdriverio/webdriverio#12429] [Community]
Hello all, 
I am trying to run my test as I usually run it but I am getting this message 

> webdriverio-local@1.0.0 testing
> npx wdio functional-testing/config/chrome.config.ts --cucumberOpts.tagExpression="@testing"
Execution of 25 workers started at 2024-03-04T18:35:28.488Z

And then the execution never starts or even ends.
Does someone have any idea of how can I solve it? 

Here is my chrome.config.ts, I haven´t change anything.

`import { BaseConfig} from "./wdio.conf";
const ChromeOptions: WebdriverIO.Config = {
  path: "/",
  port: 9515, // default for ChromeDriver
  services: ["chromedriver", "intercept"],
  //@ts-ignore
  chromeDriverLogs: "./logs",
};

const ChromeConfig = {
  capabilities: [
    {
      // Set maxInstances to 1 if screen recordings are enabled:
      maxInstances: parseInt(process.env.MAX_INSTANCES),
      browserName: "chrome",
      "goog:chromeOptions": {
        excludeSwitches: ["enable-logging"],
        args: ["--start-maximized"],
      },
      acceptInsecureCerts: true
    },
  ],
};
const config = Object.assign({}, BaseConfig, ChromeOptions, ChromeConfig);
export { config, ChromeOptions, ChromeConfig };`

**Comment by christian-bromann** [comment]
@CSanchez58 can you look into the wdio logs to see where things stale?
